programs/manifest/src/state/utils.rs

```diff
@@ -235,7 +242,32 @@ pub(crate) fn try_to_move_global_tokens<'a, 'info>(
             ManifestError::MissingGlobal,
             "Missing global mint",
         )?;
+
+        // Don't bother checking new vs old config. If a token has/had a non-zero
+        // fee, then we do not allow it for global.
```

Comment:
i think this can cause issues where if someone enables and disables transfer fees we run into issues. i'm not sure how to handle this exactly. good to merge and consult with spl author.

Reply:
I dont think there is an issue. Dont be extra accomodating to these scammers. Some cases that you think might work but dont, not a big deal, your fault for working with scammers